packages/shared/src/calculateAPY.ts

```diff
@@ -1,21 +1,40 @@
 import { Queries, stakers, Staker } from "@tender/shared/src/index";
 import { ProtocolName } from "./data/stakers";
 
+const YEAR_IN_SECONDS = 60 * 60 * 24 * 365;
```

Comment:
```suggestion
const YEAR_IN_SECONDS = 31536000;
```

Reply:
I prefer the former notation, easier to read, also it's top level so it's only calculated once